custom/icds_reports/models.py

```diff
@@ -862,3 +864,77 @@ def aggregate(cls, state_id, month):
             cursor.execute(helper.drop_table_query())
             cursor.execute(curr_month_query, curr_month_params)
             cursor.execute(agg_query, agg_params)
+
+
+class ChildHealthMonthly(models.Model):
```

Comment:
probably revealing my ignorance about how this all works, but where does this table get created? or was it already being created and just had no model?

Reply:
Hi @czue, sorry for late response. This table is creating in this script https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/migrations/sql_templates/create_tables_and_views.sql#L246